docs/building-from-source.md

```diff
@@ -61,9 +61,9 @@ If you check out repos manually then it is your responsibility to ensure that th
 ```
 mkdir repos
 git -C repos clone https://github.com/llvm/llvm-project.git
-git -C repos/llvm-project am -k ../../patches/llvm-project/*.patch
+git -C repos/llvm-project am -k "$(PWD)"/../patches/llvm-project/*.patch
```

Comment:
Sorry, but you've combined parts of both my suggestions into something that doesn't work. (Did you re-test it, even as a standalone command?)

`"$PWD"` means: expand the shell variable `PWD`, which the shell should already be keeping up to date with your current directory. `"${PWD}"` would also work: the braces explicitly delimit the variable name in case of ambiguity, although in this case there's no ambiguity in the first place.

`"$(pwd)"` means: run the subcommand `pwd`, and substitute its output.

But `"$(PWD)"` means run the subcommand `PWD`, and _that_ subcommand doesn't exist, because Unix is case sensitive:
```
$ echo "$(PWD)"
PWD: command not found
```

Reply:
Sorry, this is a case of works-on-my-pc, because apparently I have:
$ which PWD
/bin/PWD

Let me fix it again.

Reply:
Ah, I bet that was on a Mac? MacOS is like Unix except case insensitive (by default), so yes, if you test on a Mac, you can miss all sorts of case-related problems that show up everywhere else.

Reply:
Yes, I am on macOS.